pyqtgraph/examples/ExampleApp.py

```diff
@@ -7,6 +7,8 @@
 from collections import OrderedDict
 from functools import lru_cache
 
+import pkg_resources
```

Comment:
Can it be assumed that this is available for use?

Reply:
Not as sure as I am about pkgutil so we can use that with minimal changes instead of pkg_resources.